src/bci_build/package/__init__.py

```diff
@@ -1027,20 +1036,24 @@ def readme(self) -> str:
                 return self.extra_files["README.md"].decode("utf-8")
             return str(self.extra_files["README.md"])
 
+        # default template if no custom README.md.j2 template is provided in the package folder
+        readme_template = textwrap.dedent(f"""
+            # The {self.title} Container image
```

Comment:
Either capitalize both container and image or neither
```suggestion
            # The {self.title} Container Image
```

Reply:
@dcermak you do realize that I just moved the line around and it has been that way all the time?

Reply:
Ok, so let's not fix it then